keyboards/aki27/cocot46plus/keyboard.json

```diff
@@ -0,0 +1,152 @@
+{
+  "keyboard_name": "cocot46plus",
+  "manufacturer": "aki27",
+  "maintainer": "markstos",
+  "tags": ["unibody","ergo","corne","trackball","encoder","rgb"]
+  "usb": {
+    "device_version": "1.0.0",
+    "pid": "0x0003",
+    "vid": "0x1727"
+  },
+  "processor": "atmega32u4",
+  "bootloader": "caterina",
+  "qmk": {
+    "locking": {
+      "enabled": false,
+      "resync": false
+    }
+  },
+  "build": {
+    "lto": true
+  },
+  "features": {
+    "audio": false,
+    "backlight": false,
+    "bootmagic": true,
+    "command": false,
+    "console": false,
+    "encoder":true,
+    "extrakey": true,
+    "mousekey": true,
+    "nkro": false,
+    "oled": true,
```

Comment:
If OLEDs are enabled at the keyboard level, there also needs to be some configuration for them.

Reply:
I'll remove it from keyboard.json then.